doc/whats_new.rst

```diff
@@ -62,6 +62,16 @@ Model Selection Enhancements and API Changes
     the corresponding parameter is not applicable. Additionally a list of all
     the parameter dicts are stored at ``results_['params']``.
 
+  - **Renaming of ``n_folds`` and ``n_iter`` to ``n_splits``**
+
+    Some parameter names have changed: 
```

Comment:
I thought after [this](https://github.com/scikit-learn/scikit-learn/pull/7187#discussion_r74945171) change, this is not needed...